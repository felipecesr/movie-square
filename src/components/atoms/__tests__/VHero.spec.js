import { shallowMount } from '@vue/test-utils';
import VHero from '@/components/atoms/VHero.vue';

describe('VHero.vue', () => {
  const props = {
    alt: '299536',
    src: 'Avengers: Infinity War'
  };

  const wrapper = shallowMount(VHero, { propsData: props });

  it('renders correctly', () => {
    expect(wrapper.element).toMatchSnapshot();
  });

  it('img atrrs should be equal props', () => {
    expect(wrapper.props().src).toBe(props.src);
    expect(wrapper.props().alt).toBe(props.alt);
  });
});
