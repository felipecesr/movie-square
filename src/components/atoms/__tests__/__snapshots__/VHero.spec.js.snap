// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VHero.vue renders correctly 1`] = `
<img
  alt="299536"
  src="Avengers: Infinity War"
/>
`;
